standard/ERCs/erc-6900.md

```diff
@@ -460,58 +460,59 @@ interface IExecutionHookModule is IModule {
+- The account MUST emit `ValidationInstalled` as defined in the interface for all installed validation functions.
 
 #### Uninstallation
 
 During validation uninstallation, the account MUST correctly clear flags and other fields based on the incoming data provided by the user.
 
-- the account MUST clear all flags for the validation function, like `isGlobal`, `isSignatureValidation`, and `isUserOpValidation`.
-- the account MUST remomve all hooks and SHOULD clear hook module states by calling `onUninstall` with the user-provided data for each hook, including both pre validation hooks and execution hooks, if required by user.
-- the account MUST remove all selectors that the validation function can validate.
-- the account MUST emit `ValidationUninstalled` as defined in the interface for all uninstalled validations.
+- The account MUST clear all flags for the validation function, like `isGlobal`, `isSignatureValidation`, and `isUserOpValidation`.
+- The account MUST remove all hooks and SHOULD clear hook module states by calling `onUninstall` with the user-provided data for each hook, including both validation hooks and execution hooks, if specified by the user.
+- The account MUST clear the configuration for the selectors that the validation function can validate.
+- The account SHOULD call `onUninstall` on the validation module to clean up state if specified by the user.
+- The account MUST emit `ValidationUninstalled` as defined in the interface for all uninstalled validation functions.
 
-### Execution and Their Installation/Uninstallation
+### Execution Functions and Their Installation/Uninstallation
 
-An account can install any number of execution functions.
-An execution function selector MUST be unique in the account.
-An execution function selector MUST not conflict with native ERC-4337 and ERC-6900 functions.
+- An account can install any number of execution functions.
+- An execution function selector MUST be unique in the account.
+- An execution function selector MUST not conflict with native ERC-4337 and ERC-6900 functions.
 
 #### Installation
 
 During execution installation, the account MUST correctly set flags and other fields based on the incoming data and module manifest provided by the user.
 
-- the account MUST install all execution functions and set flags and fields as specified in the manifest.
-- the account MUST add all execution hooks as specified in the manifest.
-- the account SHOULD add all supported interfaces as specified in the manifest.
-- the account SHOULD call `onInstall` on the execution module to initialize the states if required by user.
-- the account MUST emit `ExecutionInstalled` as defined in the interface for all installed executions.
+- The account MUST install all execution functions and set flags and fields as specified in the manifest.
+- The account MUST add all execution hooks as specified in the manifest.
+- The account MUST add all supported interfaces as specified in the manifest.
```

Comment:
As discussed in the working group sync, we should make this "SHOULD" and also update the corresponding section for uninstallation.